Neo4j.Driver/Neo4j.Driver/Internal/Protocol/BoltProtocolMessageFactory.cs

```diff
@@ -85,11 +85,6 @@ internal class BoltProtocolMessageFactory : IBoltProtocolMessageFactory
         return new RunWithMetadataMessage(connection.Version, query, notificationsConfig: notificationsConfig);
     }
 
-    public TelemetryMessage NewTelemetryMessage(QueryApiType apiUsage)
```

Comment:
Has this been moved elsewhere?

Reply:
it is an unused method, there is a different NewTelemetry method

Reply:
line 167 is the correct method